template/java8/model/src/main/java/com/openfaas/model/IRequest.java

```diff
@@ -9,5 +9,7 @@
     String getBody();
     Map<String, String> getHeaders();
     String getHeader(String key);
+    String getQueryRaw();
+    Map<String, String> getQueryParameters();
```

Comment:
I'm thinking this should just be `getQuery()`?

Reply:
you mean getQueryParameters() should be changed to getQuery()?